server/db-config.js

```diff
@@ -285,7 +297,7 @@ dataBase.edit = function (table, doc, cb){
             editInvoices(doc, client, cb, done);
         } else if (table === 'contacts') {
             editContacts(doc, client, cb, done);
-        } else if (table ==='reminders') {
+        } else if (table ==='reminderer') {
```

Comment:
@benjaminlees wasn't the plan that this table would be just a test, and then you would go back to using the `reminders` table? Not sure if `reminderer` is a great name for a table in production.